modules/mobile/spec/request/claims_and_appeals_overview_request_spec.rb

```diff
@@ -276,6 +276,75 @@
         expect(response.body).to match_json_schema('claims_and_appeals_overview_response')
       end
     end
+
```

Comment:
I think this might read better if it were structured like:
- when user has access to both
- when user has access to claims
- when user has access to appeals
- when you user has access to neither